easybuild/easyconfigs/x/xtb/xtb-6.2.2-hotfix-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MesonNinja'
+
+name = 'xtb'
+version = '6.2.2-hotfix'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.chemie.uni-bonn.de/pctc/mulliken-center/grimme/software/xtb'
+description = """
+xtb - An extended tight-binding semi-empirical program package.
+"""
```

Comment:
@dquigley-warwick Don't want to be too pedantic, but this `description` is a bit ugly. Could you change it to a single line, as you do for `Meson`?